### PR TITLE
Include purchased heat and steam EF (EPA)

### PR DESCRIPTION
**Pull request by greenercode**
**Is this a new addition or an update to an existing factor?**

[x] New

**What category or categories does it affect?**

Heat and steam 

**What?**

Include purchased heat and steam EF (EPA)

**Why?**

This emission factor greatly contributes to the scientific depth of the existing category.

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -436,6 +436,7 @@ Energy,Heat and Steam,heat-and-steam-type_district,District heat and steam,kg-CO
...
 Energy,Heat and Steam,heat-and-steam-type_onsite,Onsite heat and steam,kg-CO2e/kWh,0.17073,,BEIS,2021,GB,2021/11/25,Emission intensity for onsite heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,use_phase
 Energy,Heat and Steam,heat-and-steam-type_onsite_wtt,Onsite heat and steam (WTT),kg-CO2e/kWh,0.03153,,BEIS,2021,GB,2021/11/25,Emission intensity for Well-to-Tank - onsite heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,upstream
+Energy,Heat and Steam,heat-and-steam-type_purchased,Purchased heat and steam,kg-CO2e/kWh,0.08907,,EPA,2021,US,2021/12/12,Emission intensity for steam or heat purchased per kWh of steam. These factors assume natural gas fuel is used to generate steam or heat at 80 percent thermal efficiency. Available in the EPA's GHG Emission Factors Hub.,https://www.epa.gov/climateleadership/ghg-emission-factors-hub,use_phase
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:439]: the "per kWh of steam" - could this simply be "per kWh", as steam and heat already mentioned in the sentence?

**Review by patstirling — CHANGES_REQUESTED** [review]
Nice addition! Just a small comment

**Review by patstirling — APPROVED** [review]

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -436,6 +436,7 @@ Energy,Heat and Steam,heat-and-steam-type_district,District heat and steam,kg-CO
...
 Energy,Heat and Steam,heat-and-steam-type_onsite,Onsite heat and steam,kg-CO2e/kWh,0.17073,,BEIS,2021,GB,2021/11/25,Emission intensity for onsite heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,use_phase
 Energy,Heat and Steam,heat-and-steam-type_onsite_wtt,Onsite heat and steam (WTT),kg-CO2e/kWh,0.03153,,BEIS,2021,GB,2021/11/25,Emission intensity for Well-to-Tank - onsite heat and steam. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,upstream
+Energy,Heat and Steam,heat-and-steam-type_purchased,Purchased heat and steam,kg-CO2e/kWh,0.08907,,EPA,2021,US,2021/12/12,Emission intensity for steam or heat purchased per kWh. This factor assumes natural gas fuel is used to generate steam or heat at 80 percent thermal efficiency. Available in the EPA's GHG Emission Factors Hub.,https://www.epa.gov/climateleadership/ghg-emission-factors-hub,use_phase
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:439]: Does "use-phase" refer to ng combustion?

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:439]: Looks like it does - "fuel_combustion" seems the more accurate; we'd need to understand whether there was any upstream activities (extraction, transmission) accounted for

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:439]: Agree. Let's use fuel_combustion if it is the only activity to be consistent.

**Review by maryarbab — APPROVED** [review]

**Review by maryarbab — APPROVED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**